docs/guides/viewing_aggregations.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: home
+title: Viewing aggregations
+parent: How-to guides
+nav_order: 6
+---
+
+## Viewing search aggregations
+Aggregations list the total count of search results meeting a certain criterion. For example, you may want to know how
+many results of a query are in a certain language; aggregations will provide this data as part of the results.
+
+You can review [our reference documentation for the Aggregations type](../reference/#definition-Aggregations) to see all
+aggregations provided by TIMDEX. In the example below, we will use the `source` aggregation to see how records matching
+the search term "modal jazz" are distributed across sources.
+
+
+```graphql
+{
+  search(searchterm: "modal jazz") {
+    records {
+      source
+      sourceLink
+      title
+    }
+    source {
+      docCount
+      key
+    }
+  }
+}
+```
+
+Note that all aggregations are multivalue fields that require the `docCount` and `key` subfields. If you are using the
+[GraphQL playground](https://timdex.mit.edu/playground) and do not include these, it should add them automatically.
```

Comment:
It added them for me yesterday, but I guess it may have auto-suggested them and I provided some input to accept the suggestion before I even saw it.

I share your waffliness on this one. Since we're both feeling iffy about it, I'd suggest removing it.

Reply:
Oh it totally worked with no input and it's cool, but yeah let's pull that tip out for now until we can find a way to integrate aside "tips" or something in the markdown.